src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        class="card__picture"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&Prime; Retina 5K Monoblock"
+      >
+      <h2 class="card__title title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <h3 class="card__product-code product-code">Product code: 195434</h3>
+
+      <div class="card__product-rating product-rating">
+        <div class="product-rating__stars product-rating__stars--active-4">
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+        </div>
+        <p class="product-rating__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+
+      <a
+        href="/"
+        class="card__button button"
+        data-qa="hover"
+      >
```

Comment:
```suggestion
      <a
        href="#"
        class="card__button button"
        data-qa="hover"
      >
```
Clicking on a link redirects to 404 not fount.   
In links without url it's better to use "#".

Reply:
done